packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -575,6 +578,13 @@ export const CLIENT_ERROR_CODE_TO_ERROR_PAYLOAD: Record<number, Partial<ClientEv
     category: 'other',
     fatal: true,
   },
+  12000: {
+    errorDescription: ERROR_DESCRIPTIONS.FRAUD_DETECTION,
+    category: 'expected',
+    fatal: true,
+    name: 'locus.response',
```

Comment:
What does locus.response mean again?  What else could it be?

Reply:
Not really sure. I assumed something like "this was based on a locus response".  The other places that were using it seemed similar to mine.  The default is "other".